app/containers/Settings/index.js

```diff
@@ -21,14 +22,18 @@ import * as actions from './actions';
 import * as selectors from './selectors';
 
 class Settings extends Component {
-  state = {
-    id: null,
-    modalVisible: false,
-    firstName: '',
-    lastName: '',
-    photo: null
+  constructor(props) {
+    super(props);
+    this.setModalVisible = this.setModalVisible.bind(this);
+    this.state = {
+      id: null,
+      modalVisible: false,
+      firstName: '',
+      lastName: '',
+      photo: null,
+      scalesPageToFit: true
```

Comment:
we never change this value right? if so I don't think we need to add state for this

Reply:
ok

Reply:
so we can remove the state ?

Reply:
yes.